packages/@react-spectrum/s2/stories/TagGroup.stories.tsx

```diff
@@ -50,12 +50,13 @@ export default meta;
 
 export let Example = {
   render: (args: any) => {
+    let props = {...args};
```

Comment:
changes in this file were just so that the controls would work. previously setting the `remove` would cause the story to crash